src/Altinn.Platform/Altinn.Platform.Authorization/Authorization/Services/Implementation/ActorsWrapper.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Runtime.Serialization.Json;
+using System.Threading.Tasks;
+using Altinn.Platform.Authorization.Clients;
+using Altinn.Platform.Authorization.Configuration;
+using Altinn.Platform.Authorization.Services.Interface;
+using Common.Model;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Newtonsoft.Json;
+
+namespace Altinn.Platform.Authorization.Services.Implementation
+{
+    /// <summary>
+    /// Wrapper for the actors api
+    /// </summary>
+    public class ActorsWrapper : IActor
+    {
+        private readonly GeneralSettings _generalSettings;
+        private readonly ILogger _logger;
+        private readonly ActorClient _actorClient;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ActorsWrapper"/> class
+        /// </summary>
+        /// <param name="generalSettings">the general settings</param>
+        /// <param name="logger">the logger</param>
+        /// <param name="actorClient">the client handler for actor api</param>
+        public ActorsWrapper(IOptions<GeneralSettings> generalSettings, ILogger<ActorsWrapper> logger, ActorClient actorClient)
+        {
+            _generalSettings = generalSettings.Value;
+            _logger = logger;
+            _actorClient = actorClient;
+        }
+
+        /// <inheritdoc />
+        public async Task<List<Actor>> GetActors(int userId)
+        {            
+            List<Actor> actorList;
+            var request = new HttpRequestMessage(
+                HttpMethod.Get,
+                "api/movies");
+            request.Headers.Accept.Add(new MediaTypeWithQualityHeaderValue("application/json"));
+            request.Headers.AcceptEncoding.Add(new StringWithQualityHeaderValue("gzip"));
```

Comment:
Why do we need gzip?

Reply:
https://betterexplained.com/articles/how-to-optimize-your-site-with-gzip-compression/

I thought we could try using the gzip, if it will help in speeding up our process

Reply:
Removed this, we will initiate discussion on this in tech forum and take into use at a later point